dask/dataframe/tests/test_groupby.py

```diff
@@ -3215,6 +3209,10 @@ def test_empty_partitions_with_value_counts(by):
         ],
         columns=["A", "B", "C"],
     )
+    if pyarrow_strings_enabled():
+        # have to convert before value_counts, to
+        # count pyarrow string vs objects
+        df = to_pyarrow_string(df)
```

Comment:
I think this still captures the original intent of the test (i.e. empty partitions are handed off to `_value_counts`) but also avoids the need for `pyarrow` string-specific code 

```suggestion
    df = df.convert_dtypes()
```